From 888e55d1a9a3c246f83f2111a9b15a3f4fd9ca77 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 4 Aug 2017 13:59:26 +0800
Subject: [PATCH 2326/5242] MLK-16136-4 ARM64: dts: imx8mq-evk: disable LCDIF
 by default.

commit  5b376bba7d119ba4e40b2a6c367ae574c8a7ab98 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable LCDIF by default and DCSS will be the default
display controller on imx8mq-evk board.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 33ef60a..030312c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -487,7 +487,7 @@
 };
 
 &lcdif {
-	status = "okay";
+	status = "disabled";
 	disp-dev = "mipi_dsi_northwest";
 	display = <&display0>;
 
-- 
1.7.9.5

